src/components/MapView/mapTest.html

```diff
@@ -0,0 +1,40 @@
+
+//test html
+
+<!doctype html>
+
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title> first leaflet</title>
+	<link rel="stylesheet" href="https://unpkg.com/leaflet@1.0.1/dist/leaflet.css"/>
+
+    <style>
+        #MapView{
+            position:absolute;
+            top:0;
+            bottom:0;
+            right:0;
+            left:0;
+            width:300px;
+        }
+
+    </style>
+
+
+</head>
+
+<body>
+<div id="MapView"></div>
+<script src="https://unpkg.com/leaflet@1.0.1/dist/leaflet.js"></script>
+
+<script>
```

Comment:
Any javascript you use should be in MapView.js for consistency. you can add this on MapView.js after line 16